testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/core/servlet/ServletConfigTest.java

```diff
@@ -75,7 +75,7 @@ public void testCount() throws Exception {
     public void testNullJaxb() throws Exception {
         Response response = client.target(generateURL("/my/null")).request().header("Content-Type", "application/xml")
                 .post(Entity.text(""));
-        Assert.assertEquals(HttpResponseCodes.SC_BAD_REQUEST, response.getStatus());
+        Assert.assertEquals(HttpResponseCodes.SC_UNSUPPORTED_MEDIA_TYPE, response.getStatus());
```

Comment:
This was changed in #2794 ([RESTEASY-2874](https://issues.redhat.com/browse/RESTEASY-2874)) and reverted back here. This should likely not have been changed.

The test itself might be incorrect, but it shouldn't have been changed.